span-stacktrace/src/test/java/io/opentelemetry/contrib/stacktrace/StackTraceAutoConfigTest.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.contrib.stacktrace;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import io.opentelemetry.sdk.autoconfigure.spi.internal.DefaultConfigProperties;
+import io.opentelemetry.sdk.trace.ReadableSpan;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.function.Predicate;
+import org.junit.jupiter.api.Test;
+
+public class StackTraceAutoConfigTest {
+
+  @Test
```

Comment:
I think we could improve the coverage of this test if you instead test on the level of the `TracerProviderCustomizer` using : This way you could record calls to `addSpanProcessor` and introspect the added processor for the correct configuration (or ensuring that no processor is added at all).

Reply:
I've done that in 3072467832520b0ed37831ea6758b6c999a27e1c but in the other test class as it's more an "end-to-end" setup with a complete SDK, this one remains more a unit-test.